cmd/subnetcmd/upgradecmd/print_upgrade.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package upgradecmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/utils/storage"
+	"github.com/spf13/cobra"
+)
+
+// avalanche subnet upgrade import
+func newUpgradePrintCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "print [subnetName]",
```

Comment:
Do we like `print` or `describe`?

Reply:
In this case I think `print` is correct, as we are just printing the actual file contents to the screen.